content/docs/contributing/contributing_changes.md

```diff
@@ -159,7 +152,12 @@ unexpectedly.
 
 ## The Review Process
 
-Once your pull request has been submitted the handbook maintainers will be
-prompted to make a review. Each pull request requires **at least one** approval from
-the maintainers before merging. If a reviewer requests any changes or makes any
-comments, these **must** be resolved before the pull request can be merged.
+Currently reviews are not required and pull requests may be merged by anyone
+with write access once all required CI checks pass. If you like you can enable
+[auto-merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)
+so that the merge happens automatically when this is the case.
+
+However, if you'd like your changes reviewed you should just leave your pull
+request open: the handbook maintainers will be automatically prompted to make a
+review. If a reviewer then requests any changes or makes any comments, these
+should be resolved before the pull request is merged.
```

Comment:
```suggestion
However, if you'd like your changes reviewed you can suggest reviewers when, or after, opening your pull request. 
If a reviewer then requests any changes or makes any comments, these should be resolved before the pull request is merged.
```

At the moment, there is no maintainer group and no one is automatically prompted to review (but we can of course change this if we have maintainers).

Reply:
Thanks for the suggested change; I've incorporated it in.